src/components/App.js

```diff
@@ -1,9 +1,16 @@
+import { Route, Switch } from 'react-router-dom';
+import About from './About';
 import Home from './Home';
 
 function App() {
   return (
     <div className="page">
-      <Home />
+      <Switch>
+        <Route exact path="/">
```

Comment:
if i not mistake you can try use Route with close brace, so like that `<Route exact path="/" component={<Home />}/>`